src/webapi/tests/integ.rs

```diff
@@ -0,0 +1,60 @@
+use once_cell::sync::Lazy;
+use std::{net::TcpListener, str::FromStr};
+use tracing::Level;
+use webapi::{
+    startup::run,
+    telemetry::{get_subscriber, init_subscriber},
+};
+
+// Ensure that the `tracing` stack is only initialised once using `once_cell`
```

Comment:
Please be consistent with spelling. You've used `initialize` elsewhere, here you wrote `initialised`.